<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from kaka.xacro                     | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<robot name="kaka" xmlns:controller="http://playerstage.sourceforge.net/gazebo/xmlschema/#controller" xmlns:interface="http://playerstage.sourceforge.net/gazebo/xmlschema/#interface" xmlns:sensor="http://playerstage.sourceforge.net/gazebo/xmlschema/#sensor" xmlns:xacro="http://playerstage.sourceforge.net/gazebo/xmlschema/#interface">
  <!-- end of macro define -->
  <!-- end of macro define -->
  <!-- end of macro define -->
  <link name="base_link">
    <visual>
      <geometry>
        <cylinder length="0.04" radius="0.04"/>
      </geometry>
      <material name="blue">
        <color rgba="0 0 .8 1"/>
      </material>
    </visual>
    <collision>
      <geometry>
        <cylinder length="0.04" radius="0.04"/>
      </geometry>
    </collision>
    <inertial>
      <mass value="0.2"/>
      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
    </inertial>
  </link>
  <link name="right_arm">
    <visual>
      <geometry>
        <box size="0.17 .01 .01"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <material name="white">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <geometry>
        <box size="0.17 .01 .01"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
    </collision>
    <inertial>
      <mass value="0.07"/>
      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
    </inertial>
  </link>
  <joint name="base_to_right_arm" type="revolute">
    <axis xyz="0 -1 0"/>
    <limit effort="1000.0" lower="-1.57075" upper="1.57075" velocity="0.5"/>
    <parent link="base_link"/>
    <child link="right_arm"/>
    <origin xyz="0 -0.06  0"/>
  </joint>
  <link name="right_disk">
    <visual>
      <geometry>
        <cylinder length="0.001" radius="0.1"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <material name="white">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <geometry>
        <cylinder length="0.001" radius="0.1"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
    </collision>
    <inertial>
      <mass value="0.01"/>
      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
    </inertial>
  </link>
  <joint name="right_disk_arm_joint" type="fixed">
    <parent link="right_arm"/>
    <child link="right_disk"/>
    <origin xyz="0 -0.085 0.019"/>
  </joint>
  <link name="left_arm">
    <visual>
      <geometry>
        <box size="0.17 .01 .01"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <material name="white">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <geometry>
        <box size="0.17 .01 .01"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
    </collision>
    <inertial>
      <mass value="0.07"/>
      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
    </inertial>
  </link>
  <joint name="base_to_left_arm" type="revolute">
    <axis xyz="0 1 0"/>
    <limit effort="1000.0" lower="-1.57075" upper="1.57075" velocity="0.5"/>
    <parent link="base_link"/>
    <child link="left_arm"/>
    <origin xyz="0 0.06  0"/>
  </joint>
  <link name="left_disk">
    <visual>
      <geometry>
        <cylinder length="0.001" radius="0.1"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <material name="white">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <geometry>
        <cylinder length="0.001" radius="0.1"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
    </collision>
    <inertial>
      <mass value="0.01"/>
      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
    </inertial>
  </link>
  <joint name="left_disk_arm_joint" type="fixed">
    <parent link="left_arm"/>
    <child link="left_disk"/>
    <origin xyz="0 0.085 0.019"/>
  </joint>
</robot>

